lib/src/fileset.rs

```diff
@@ -473,9 +485,11 @@ pub fn parse(
     diagnostics: &mut FilesetDiagnostics,
     text: &str,
     path_converter: &RepoPathUiConverter,
+    aliases_map: &FilesetAliasesMap,
 ) -> FilesetParseResult<FilesetExpression> {
     let node = fileset_parser::parse_program(text)?;
     // TODO: add basic tree substitution pass to eliminate redundant expressions
+    let node = dsl_util::expand_aliases(node, aliases_map)?;
```

Comment:
nit: in order to write parsing tests, it might be better to add `fileset_parser::parse(text, aliases_map)` (and make parse_program private.)